common/src/main/java/org/opensearch/ml/common/ActionType.java

```diff
@@ -0,0 +1,16 @@
+package org.opensearch.ml.common;
+
+import java.util.Locale;
+
+public enum ActionType {
```

Comment:
Why add another ActionType ? Can we reuse `ActionType` in `ConnectorAction` class